cirrus/server/tests/test_sdk.py

```diff
@@ -1,13 +1,52 @@
+import json
 import unittest
 
 from cirrus_sdk import NimbusError
+from parameterized import parameterized
 
+from cirrus.main import sdk
```

Comment:
This is the same pattern again.  Don't initialize stateful objects in the global namespace at all.  If you need an instance of `SDK` in the tests, initialize it inside the `setUp` method.